content/collaborators/communities-of-place/DirectED Abuja Hub/Hub Team.md

```
@@ -0,0 +1,30 @@
+The Hub Team provides structure, coherence, and support for all teams within the DirectED Abuja Hub. Roles are adaptive and collaborative, following the principles of distributed leadership and place-based coordination.
+
+**Emerald Akhaumere**
+---
+
+**Hub Lead / Hub Communications**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)